src/main/java/dinkplugin/notifiers/DeathNotifier.java

```diff
@@ -127,16 +139,26 @@ protected String getWebhookUrl() {
     }
 
     public void init() {
-        setIgnoredRegions(config.deathIgnoredRegions());
+        clientThread.invoke(() -> setIgnoredRegions(config.deathIgnoredRegions()));
     }
 
     public void reset() {
-        setIgnoredRegions(null);
+        clientThread.invoke(() -> {
+            setIgnoredRegions(null);
+            this.resurrectedAt = null;
+        });
     }
 
     public void onConfigChanged(String key, String value) {
         if ("deathIgnoredRegions".equals(key)) {
-            setIgnoredRegions(value);
+            clientThread.invoke(() -> setIgnoredRegions(value));
+        }
+    }
+
+    public void onVarbit(VarbitChanged event) {
+        if (event.getValue() != 1) return;
+        if (event.getVarbitId() == KARAMJA_RESURRECTION_USED || event.getVarbitId() == WESTERN_RESURRECTION_USED) {
+            this.resurrectedAt = Instant.now();
```

Comment:
if this approach works, we could use game message instead of undocumented varbits if desired

Reply:
>undocumented varbits 

wrong repo